Modules/Files/src/Files.Infrastructure/Persistence/Database/EntityTypeConfigurations/FileEntityTypeConfiguration.cs

```diff
@@ -13,6 +13,8 @@ public override void Configure(EntityTypeBuilder<File> builder)
 
         builder.ToTable("FileMetadata");
 
+        builder.Property(m => m.OwnershipToken).IsRequired();
```

Comment:
```suggestion
        builder.Property(m => m.OwnershipToken);
```

It may even be possible to remove this line completely